notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb

```diff
@@ -1224,7 +1224,11 @@
    "id": "162968cb-3f83-4aab-8c76-e557e5a21cc0",
```

Comment:
<p>@dianagroch can you please check this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdenkas/human-tafazzin-variants-database/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p><strong>row 273:</strong></p><p><strong>Lift over: hg19:</strong><a href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&amp;position=chrX:153648426-153648429" target="_blank">X:153648427</a> <strong>hg38:</strong><a href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg38&amp;position=chrX:154420087-154420090" target="_blank">chrX:154420088</a></p><p>should be C in this position , c.640C&gt; T, p.H214Y - correct</p><p> </p><p>yes, just problem with conversion</p>